Index: kde-telepathy-text-ui/cmake/modules/FindTelepathyLoggerQt4.cmake
===================================================================
--- kde-telepathy-text-ui.orig/cmake/modules/FindTelepathyLoggerQt4.cmake	2012-11-28 00:10:21.936607799 +0530
+++ kde-telepathy-text-ui/cmake/modules/FindTelepathyLoggerQt4.cmake	2012-11-29 00:04:39.198474099 +0530
@@ -19,24 +19,13 @@
 set(TELEPATHY_LOGGER_QT4_LIBRARIES ${TELEPATHY_LOGGER_QT4_LIBRARY} )
 set(TELEPATHY_LOGGER_QT4_INCLUDE_DIRS ${TELEPATHY_LOGGER_QT4_INCLUDE_DIR} ${TELEPATHY_LOGGER_QT4_INCLUDE_DIR}/telepathy-logger-0.2 )
 
-find_package(TelepathyLogger)
-find_package(TelepathyGlib)
-find_package(GObject)
-find_package(GIO)
-find_package(GLIB2)
 find_package(QtGLib)
 
 list(APPEND TELEPATHY_LOGGER_QT4_LIBRARIES
-    ${TELEPATHY_LOGGER_LIBRARIES}
-    ${TELEPATHY_GLIB_LIBRARIES}
-    ${GLIB2_LIBRARIES}
-    ${GOBJECT_LIBRARIES}
     ${QTGLIB_LIBRARIES}
-    ${GIO_LIBRARIES}
 )
 
 list(APPEND TELEPATHY_LOGGER_QT4_INCLUDE_DIRS
-    ${GLIB2_INCLUDE_DIR}
     ${QTGLIB_INCLUDE_DIR}
 )
 
