--- a/lib/logmanager.cpp
+++ b/lib/logmanager.cpp
@@ -33,9 +33,6 @@
 #include <TelepathyLoggerQt4/TextEvent>
 #include <TelepathyLoggerQt4/CallEvent>
 #include <TelepathyLoggerQt4/LogManager>
-
-#include <glib-object.h>
-#include <QGlib/Init>
 #endif
 
 #include <TelepathyQt/Types>
@@ -48,8 +45,6 @@
     m_fetchAmount(10)
 {
 #ifdef TELEPATHY_LOGGER_QT4_FOUND
-    g_type_init();
-    QGlib::init();
     Tpl::init();
 
     m_logManager = Tpl::LogManager::instance();
--- a/logviewer/log-viewer.cpp
+++ b/logviewer/log-viewer.cpp
@@ -28,9 +28,6 @@
 #include <TelepathyLoggerQt4/Entity>
 #include <TelepathyLoggerQt4/LogManager>
 
-#include <glib-object.h>
-#include <QGlib/Init>
-
 #include <QSortFilterProxyModel>
 
 #include "entity-model.h"
@@ -42,8 +39,6 @@
     ui->setupUi(this);
     setWindowIcon(KIcon(QLatin1String("documentation")));
     Tp::registerTypes();
-    g_type_init();
-    QGlib::init(); //are these 4 really needed?
     Tpl::init();
 
     m_accountManager = Tp::AccountManager::create();
