Index: kde-telepathy-text-ui/CMakeLists.txt
===================================================================
--- kde-telepathy-text-ui.orig/CMakeLists.txt	2012-11-28 00:11:00.008612766 +0530
+++ kde-telepathy-text-ui/CMakeLists.txt	2012-11-29 00:40:14.366838008 +0530
@@ -48,7 +48,6 @@
 
 
 add_subdirectory(filters)
-add_subdirectory(tests)
 
 find_package(Msgfmt REQUIRED)
 find_package(Gettext REQUIRED)
